### PR TITLE
added light/dark mode

### PR DESCRIPTION
**Pull request by serabaker**
video of dark-mode toggle on the navbar

https://user-images.githubusercontent.com/69497484/202864962-51c9add5-c1f9-4e12-b0ed-bf75c09a116e.mov

### CODE REVIEW

File: src/components/lightOrDarkMode.jsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+import Brightness7Icon from "@mui/icons-material/Brightness7";
+import { CssBaseline } from "@mui/material";
+
+const ColorModeContext = React.createContext({ toggleColorMode: () => {} });
+
+function LightOrDarkMode() {
+  const theme = useTheme();
+  const colorMode = React.useContext(ColorModeContext);
+  return (
+    <Box
```
Comment (adriannaeguevarra) [inline src/components/lightOrDarkMode.jsx]: I think the Box is going to take over the DOM. I tried to use this within the Authenticator component and I expect to see a signout button and this toggle. However, what I ended up seeing is that the signout button disappears and the toggle button is the only thing rendered.

The functionality works. However, lets make sure that we can use it within the Authenticator component as that is wat is going to be the Higher order Component

**Review by adriannaeguevarra — CHANGES_REQUESTED** [review]
Great job on getting this in. There is a small concern that I addressed via a comment. Once that is addressed, this PR looks good!

File: src/components/lightOrDarkMode.jsx (hunk 1 above)
Comment (serabaker) [reply src/components/lightOrDarkMode.jsx]: I removed the Box from the component and the dark mode toggle is on the navbar for now.

**Review by adriannaeguevarra — APPROVED** [review]
Looks good

**Pull request merged**